modules/orchestrator/src/main/java/org/eclipse/xpanse/orchestrator/utils/OpenApiUtil.java

```diff
@@ -253,8 +253,8 @@ private String getApiDocsJson(RegisterServiceEntity registerService) {
         if (Objects.isNull(registerService)) {
             return StringUtils.EMPTY;
         }
-        // version of OCL.
-        String oclVersion = registerService.getOcl().getVersion();
+        // version of the service.
+        String oclVersion = registerService.getVersion();
```

Comment:
shall we rename the `oclVersion` to `serviceVersion` at the same time?

Reply:
yes makes sense. I oversaw that. Fixed now.